### PR TITLE
Deprecate passing a tuple for gr.Code value

### PR DESCRIPTION
**Pull request by freddyaboulton**
## Description

From #9122

## 🎯 PRs Should Target Issues

Before your create a PR, please check to see if there is [an existing issue](https://github.com/gradio-app/gradio/issues) for this change. If not, please create an issue before you create this PR, unless the fix is very small. 

Not adhering to this guideline will result in the PR being closed. 

## Tests

1. PRs will only be merged if tests pass on CI. To run the tests locally, please set up [your Gradio environment locally](https://github.com/gradio-app/gradio/blob/main/CONTRIBUTING.md) and run the tests: `bash scripts/run_all_tests.sh`

2. You may need to run the linters: `bash scripts/format_backend.sh` and `bash scripts/format_frontend.sh`

### CODE REVIEW

File: gradio/components/code.py [hunk 1]
```diff
@@ -155,20 +154,16 @@ def preprocess(self, payload: str | None) -> str | None:
...
-    def postprocess(self, value: tuple[str] | str | None) -> None | str:
+    def postprocess(self, value: str | None) -> None | str:
         """
         Parameters:
-            value: Expects a `str` of code or a single-element `tuple`: (filepath,) with the `str` path to a file containing the code.
+            value: Expects a `str` of code.
         Returns:
             Returns the code as a `str`.
         """
         if value is None:
```
Comment (abidlabs) [inline gradio/components/code.py:164]: If someone does pass in a tuple, can we raise an informative exception telling them this is no longer supported in `gr.Code`?

**Review by abidlabs — APPROVED** [review]
thanks @freddyaboulton, lgtm with one suggestion above!

**Review by aliabd — APPROVED** [review]
lgtm but we need to change the signature on the return function in the docs (those are still hardcoded, I'll fix that soon) https://github.com/gradio-app/gradio/blob/main/js/_website/src/lib/templates/gradio/03_components/code.svx#L61

**Comment by freddyaboulton** [comment]
Thanks all!

**Pull request merged**